identifiers/country-jp/multi_seat_constituencies.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:jp/block:chugoku,Chugoku

```

Comment:
considering making the type for the block districts have some consistency with the single member districts.

For example: If the single member districts are `ed` then consider making this `block_ed`